### PR TITLE
Popup in popup

### PR DESCRIPTION
**Pull request by exdis**

### CODE REVIEW

File: client/views/popup.js [hunk 1]
```diff
@@ -69,6 +69,11 @@ class Popup {
         this.el = document.createElement('div');
         this.el.className = 'discovery-view-popup';
 
+        Popup.els = Popup.els || new Set();
+        Popup.els.add(this.el);
```
Comment (lahmatiy) [inline client/views/popup.js]: Why we need this for?

File: client/views/popup.js [hunk 2]
```diff
@@ -69,6 +69,11 @@ class Popup {
...
+
+        Popup.elQueue = Popup.elQueue || new Set();
```
Comment (lahmatiy) [inline client/views/popup.js]: Is there a reason to make it public? Maybe local variable instead?

File: client/views/popup.js [hunk 3]
```diff
@@ -119,6 +124,12 @@ class Popup {
         const availWidthLeft = box.left - viewport.right - 3;
         const availWidthRight = viewport.right - box.left - 3;
 
+        [...Popup.els]
```
Comment (lahmatiy) [inline client/views/popup.js]: Not sure I understand what happens here...

Comment (exdis) [reply client/views/popup.js]: Yep, that was experiments. Forgot to remove.

File: client/views/popup.js (hunk 2 above)
Comment (exdis) [reply client/views/popup.js]: Maybe it does make sense. I'll change this.

**Pull request merged**